lib/screens/login/login.dart

```diff
@@ -141,7 +141,7 @@ class LoginScreen extends StatelessWidget {
                                   TextStyle(fontSize: 20 * heightFactor, color: C.secondaryColor),
                             ),
                             onTap: () {
-                              //TODO: Forgot Password Route
+                              Navigator.pushReplacementNamed(context, S.routeForgotPassword);
```

Comment:
There will be no use of the back button if you replace the route.
```suggestion
                              Navigator.pushNamed(context, S.routeForgotPassword);
```